### DISCUSSION TITLE
Vaillant aguaFLOW exclusive VPM

### DISCUSSION THREAD

**Discussion by Hendrik-HOB** [john30/ebusd#648] [Q&A]
Hi,
I try to control the circulation pump of my fresh water station.
The possible settings in the integrated controller are: Off, Eco, Comfort.
As far as I can see, the option is not yet included in the configuration files.
I am not very good in decoding all the messages.
Here is my info:

_localhost: i
version: ebusd 22.3.p20220508
device: /dev/ttyebus
signal: acquired
symbol rate: 48
max symbol rate: 167
min arbitration micros: 1
max arbitration micros: 70
min symbol latency: 3
max symbol latency: 8
reconnects: 0
masters: 6
messages: 840
conditional: 113
poll: 0
update: 10
address 00: master #1
address 03: master #11
address 05: slave #1, scanned "MF=Vaillant;ID=VR920;SW=2602;HW=5703"
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0307;HW=0403", loaded "vaillant/08.hmu.csv"
address 0a: slave, scanned "MF=Vaillant;ID=PMW01;SW=0206;HW=8302", loaded "vaillant/0a.pmw.hwc.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=70000;SW=0614;HW=6903", loaded "vaillant/15.700.csv"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0104;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0307;HW=0403"
address e0: slave, scanned "MF=Vaillant;ID=OMU00;SW=0131;HW=5202", loaded "vaillant/e0.omu.csv"
address e1: slave, scanned "MF=Vaillant;ID=OMU00;SW=0131;HW=5202", loaded "vaillant/e1.omu.1.csv"
address f7: master #20
address fc: slave #20, scanned "MF=Vaillant;ID=PMW01;SW=0206;HW=8302"_

and the result of a listen -U and grab result (mode changed from comfort to off, from off to eco and from eco to comfort):

_localhost: grab result
1008b5110100 / 09a70210000000000100 = 1
1076b5040100 / 0a03531117190901226010 = 1
1076b5110101 / 09ff506010ffff0000ff = 2
10feb505025c00 = 1
1008b507020900 / 02f202 = 1
1076b512030f0201 / 0700800000800f05 = 2
1076b51303040d00 / 02ffff = 1
f715b51303066400 / 00 = 1
1076b51009000000ffffff010000 / 0101 = 2_

I hope someone can "read" this and we can extend the configuration file.

Hendrik

**Comment by Alex-joomla** [comment]
@Hendrik-HOB did you manage it?
